tests/dft-custom-mgga/ref_outputs/g09_m06l.ref

```diff
@@ -0,0 +1,240 @@
+                 (T2G)
+ The electronic state is 1-A1G.
+ Alpha  occ. eigenvalues --  -30.75385  -1.41240  -0.48385  -0.48385  -0.48385
+ Alpha virt. eigenvalues --    1.35999   1.35999   1.35999   1.79631   4.73415
+ Alpha virt. eigenvalues --    4.73415   4.73415   4.73415   4.73415
+          Condensed to atoms (all electrons):
+               1
+     1  Ne  10.000000
+ Mulliken charges:
+               1
+     1  Ne   0.000000
+ Sum of Mulliken charges =   0.00000
+ Mulliken charges with hydrogens summed into heavy atoms:
+               1
+     1  Ne   0.000000
+ Electronic spatial extent (au):  <R**2>=              9.0979
+ Charge=              0.0000 electrons
+ Dipole moment (field-independent basis, Debye):
+    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
+ Quadrupole moment (field-independent basis, Debye-Ang):
+   XX=             -4.0790   YY=             -4.0790   ZZ=             -4.0790
+   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
+ Traceless Quadrupole moment (field-independent basis, Debye-Ang):
+   XX=              0.0000   YY=              0.0000   ZZ=              0.0000
+   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
+ Octapole moment (field-independent basis, Debye-Ang**2):
+  XXX=              0.0000  YYY=              0.0000  ZZZ=              0.0000  XYY=              0.0000
+  XXY=              0.0000  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
+  YYZ=              0.0000  XYZ=              0.0000
+ Hexadecapole moment (field-independent basis, Debye-Ang**3):
+ XXXX=             -1.7155 YYYY=             -1.7155 ZZZZ=             -1.7155 XXXY=              0.0000
+ XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
+ ZZZY=              0.0000 XXYY=             -0.5718 XXZZ=             -0.5718 YYZZ=             -0.5718
+ XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
+ N-N= 0.000000000000D+00 E-N=-3.109601378983D+02  KE= 1.283922808895D+02
+ Symmetry AG   KE= 1.029769585924D+02
+ Symmetry B1G  KE= 1.037123601901D-61
+ Symmetry B2G  KE= 8.219184344868D-62
+ Symmetry B3G  KE= 1.352852971413D-61
+ Symmetry AU   KE= 0.000000000000D+00
+ Symmetry B1U  KE= 8.471774099030D+00
+ Symmetry B2U  KE= 8.471774099030D+00
+ Symmetry B3U  KE= 8.471774099030D+00
+ 1\1\GINC-SILVER-NODE17\SP\RM06L\CC-pVDZ\Ne1\KRAUS\01-Feb-2018\0\\# M06
+ L/cc-pvdz 5D SP int=(grid=99770)\\Ne\\0,1\Ne,0,0.,0.,0.\\Version=AS64L
+ -G09RevD.01\State=1-A1G\HF=-128.9031949\RMSD=4.624e-10\Dipole=0.,0.,0.
+ \Quadrupole=0.,0.,0.,0.,0.,0.\PG=OH [O(Ne1)]\\@
+
+
+ THE THINKER DIES, BUT HIS THOUGHTS ARE BEYOND THE 
+ REACH OF DESTRUCTION.  MEN ARE MORTAL; BUT IDEAS 
+ ARE IMMORTAL.
+                              -- WALTER LIPPMANN
+ Job cpu time:       0 days  0 hours  0 minutes 14.0 seconds.
```

Comment:
This program can be sensitive. It'd be good to remove the "Job cpu time" lines from the output files. And you may want to zip these files. And if you're not the license holder, you may want to make sure the license holder is ok with posting the files.